controller/controller.go

```diff
@@ -1439,6 +1439,9 @@ func (ctrl *ProvisionController) provisionClaimOperation(ctx context.Context, cl
 	if err := ctrl.volumeStore.StoreVolume(claim, volume); err != nil {
 		return ProvisioningFinished, err
 	}
+	if err = ctrl.volumes.Add(volume.Name); err != nil {
+		utilruntime.HandleError(err)
+	}
```

Comment:
This `Add` fails all the time with:
```
couldn't create key for object pvc-17e4c2d0-114d-4260-b587-c39a86c8dfe5: object has no meta: object does not implement the Object interfaces
```

You need to add the `volume` object, not its name...

Reply:
Fix is in https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/pull/92